tests/opensource-suites/dash/selectors/selectorWithInvalidField.test.ts

```diff
@@ -0,0 +1,30 @@
+import {openTestPage, slct} from '../../../utils';
+import datalensTest from '../../../utils/playwright/globalTestDefinition';
+
+import {COMMON_SELECTORS} from '../../../utils/constants';
+import {DashUrls} from '../../../constants/test-entities/dash';
+import DashboardPage from '../../../page-objects/dashboard/DashboardPage';
+import {DashKitOverlayMenuQa, DialogControlQa, SelectQa} from '../../../../src/shared';
+
+datalensTest.describe('Dashboards - Selector behavior with invalid field types', () => {
+    datalensTest(
+        'Should successfully open edit modal when attempting to edit existing selector with incompatible field type',
+        async ({page}) => {
+            await openTestPage(page, DashUrls.DashboardWithInvalidSelector);
+            const dashboardPage = new DashboardPage({page});
+            dashboardPage.forceEnterEditMode();
```

Comment:
Bad way for future. Now you have created this dash only for your test for only one case. But it could be used some time later for other test or test case. So you may receive lot's of flaps for parallel tests runing. 
We use pattern that we copy dash in before section and deleting this copy in the after section. So that we have untouched dash without any chance of mutation or using it in parallel test case.